sources/Valkey.Glide/BaseClient.GeospatialCommands.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+using Valkey.Glide.Commands;
+using Valkey.Glide.Internals;
+
+namespace Valkey.Glide;
+
+public abstract partial class BaseClient : IGeospatialCommands
+{
+    /// <inheritdoc/>
+    public async Task<bool> GeoAddAsync(ValkeyKey key, double longitude, double latitude, ValkeyValue member, CommandFlags flags = CommandFlags.None)
+    {
+        return await GeoAddAsync(key, new GeoEntry(longitude, latitude, member), flags).ConfigureAwait(false);
+    }
+
+    /// <inheritdoc/>
+    public async Task<bool> GeoAddAsync(ValkeyKey key, GeoEntry value, CommandFlags flags = CommandFlags.None)
+    {
+        Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
```

Comment:
These exception messages should be implemented as a const. Should be able to use the same const across all command implementations not just Geo commands.

Reply:
https://github.com/valkey-io/valkey-glide-csharp/issues/112